### ISSUE TITLE
calling Azure Function from Static Web App

### ISSUE THREAD

**Issue by mizrael** [Azure/static-web-apps#445]
hi All,
I've created a Blazor Wasm project and I've deployed it as Static Web App. The Azure Portal automatically created the GitHub workflow for me, and everything worked fine.

Now I'm trying to add a Function App. I've created a .NET 5 isolated app and updated the deployment configuration to point to the right folder. I can see the Function showing up in the "Functions" blade. So far so good.

Now my question: is there a way to run/debug the whole system on localhost and call the Functions from the Blazor client? I've seen some [documentation](https://docs.microsoft.com/en-us/azure/static-web-apps/add-api?tabs=vanilla-javascript) but it's using Nodejs for the backend.

**Comment by anthonychu** [comment]
.NET 5 isolated function apps are not supported at this time if you use the function app that is built into Static Web Apps. Can you confirm that it is indeed a .NET 5 app?

As for local debugging, you would following the same steps as indicated [here](https://docs.microsoft.com/en-us/azure/static-web-apps/add-api?tabs=react#run-the-frontend-and-api-locally). Instead of `npm run build`, run `dotnet publish` and pass the CLI the generated wwwroot folder.

**Comment by mizrael** [comment]
yes, my bad. I initially deployed a .NET 3.1 version, which worked fine. Then updated it to 5 locally and "forgot" to push the changes. Too much excitement, things were going too well :D

I'll try the debugging instruction as suggested and keep you posted. Thanks!

**Comment by anthonychu** [comment]
We're investigating supporting newer .NET versions. Tracking that here: #375 . No ETA but we're hoping to have this ability before .NET 6 is GA.

**Comment by thomasgauvin** [comment]
Closing this issue, SWA currently supports .NET Core 3.1, .NET 6.0 in-process, .NET 6.0 isolated, .NET 7.0 isolated, refer to this table to all runtimes for APIs https://learn.microsoft.com/en-us/azure/static-web-apps/languages-runtimes#api
